tests/test_counting_hash.py

```diff
@@ -323,7 +323,10 @@ def test_save_load():
     hi.save(savepath)
 
     ht = khmer.CountingHash(12, sizes)
-    ht.load(savepath)
+    try:
```

Comment:
This test errored during development. I make tests fail instead of erroring when it happens to me.